packages/target-ethers-v5/src/codegen/types.ts

```diff
@@ -0,0 +1,85 @@
+import { EvmType, EvmOutputType, TupleType, AbiParameter, AbiOutputParameter } from 'typechain'
+
+export interface GenerateFunctionOptions {
+  returnResultObject?: boolean
+  isStaticCall?: boolean
+}
+
```

Comment:
I'd move this interface to the `functions.ts` file, since it's only used there and doesn't have to be exported. (this file name `types.ts` refers to handling generation for solidity types, not TS types for this package ;) )

Reply:
Very much thanks for the review. This makes sense and I'll make this change.